pyodide_build/io.py

```diff
@@ -127,10 +127,12 @@ def parse_package_config(path: Path, check: bool = True) -> Dict[str, Any]:
     """
     # Import yaml here because pywasmcross needs to run in the built native
     # Python, which won't have PyYAML
```

Comment:
Comment needs updating

Reply:
I don't really understand this comment, I guess I'll just delete it...

We could definitely improve the situation with build dependencies. Probably we should at least add a suggestion to
https://pyodide.org/en/0.17.0a2/development/building-from-sources.html#build-using-make
to look in the Dockerfile for the guaranteed up to date build dependencies?